packages/app/src/runner/events/pausing.ts

```diff
@@ -1,12 +1,18 @@
-export const handlePausing = (getCypress, reporterBus) => {
-  const Cypress = getCypress()
-  // tracks whether the cy.pause() was called from the primary driver
-  // (value === null) or from a cross-origin spec bridge (value is the origin
-  // matching that spec bridge)
-  let sendEventsToOrigin = null
+// tracks whether the cy.pause() was called from the primary driver
+// (value === null) or from a cross-origin spec bridge (value is the origin
 
-  reporterBus.on('runner:next', () => {
-    const Cypress = getCypress()
+import EventEmitter from 'events'
+
+// matching that spec bridge)
+let sendEventsToOrigin: string | null = null
+
+type GetCypressFunction = () => any
```

Comment:
are we able to actually type this to the Cypress namespace?

Reply:
updated [2f76c25](https://github.com/cypress-io/cypress/pull/31596/commits/2f76c25731a0884d2f237e76f337557e4cbf9c5c)